_docs-sources/faq/index.md

```diff
@@ -0,0 +1,66 @@
+import Card from "/src/components/Card"
+import Grid from "/src/components/Grid"
+
+# Gruntwork FAQs
+
+Get answers to frequently asked questions, organized by category.
+
+## Categories
```

Comment:
I'd propose cutting this header, as it doesn't seem to add much.

Reply:
Done in 4d9ea13